maker-frontend/src/MakerApp.tsx

```diff
@@ -137,12 +154,13 @@ export default function App() {
 
                         <GridItem colSpan={2} textAlign="center">
                             <Button
-                                disabled={isCreatingNewCfdOrder || orderPrice === "0"}
+                                disabled={isCreatingNewCfdOrder || shortPrice === "0"}
                                 variant={"solid"}
                                 colorScheme={"blue"}
                                 onClick={() => {
                                     let payload: CfdSellOrderPayload = {
-                                        price: Number.parseFloat(orderPrice),
+                                        price: Number.parseFloat(shortPrice),
```

Comment:
I think the field is called: `price_short`

Reply:
not in the old endpoint! I'll be doing a follow-up on top of this that uses the new endpoint (same with the `funding_rate`...)

Reply:
ah, got it. 👍